docs/tools/automated-testing/overview.md

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
No need to read this page again if you already read it in the other pull request, no changes here